tests/test_runtime/test_config.py

```diff
@@ -93,7 +92,16 @@ def test_config_build_detector():
         detector = build_detector(config_mod.model)
         assert detector is not None
 
-        _check_numclasscheckhook(detector, config_mod)
+        # Check whether NumClassCheckHook is used.
+        custom_hooks = config_mod.get('custom_hooks', [])
+        assert custom_hooks is None or isinstance(custom_hooks, list)
```

Comment:
unnecessary assertion for None

Reply:
`custom_hooks` is possibly set to `None`.